Sources/WalletConnectUtils/RPCHistory.swift

```diff
@@ -0,0 +1,67 @@
+import JSONRPC
+
+public final class RPCHistory {
```

Comment:
I wonder how new history can affect existing integrations 🤔
any thoughts?

Reply:
Currently, the history is being used just as a means of message deduplication, so the worst case scenario risk is the SDK receiving a duplicate `Subscription` once, which looks like a small trade-off to migrate to a history that can track all requests (the current implementation is generic, so it's bound to a single type of params). What do you think?

Reply:
Also, we still have the transition to version 1.0.0 in the future as an opportunity to include all breaking changes that we need.

Reply:
I think we want to avoid any breaking changes that could affect apps on production. But currently I do not think it will be very dangerous. Anyway it's worth to test it before merging.